gadgets/bpfstats/README.mdx

```diff
@@ -0,0 +1,314 @@
+        server side. For the `bpfstats` gadget we ran above, you
+        can find the metrics under the `bpfstats` scope:
+
+        ```bash
+        $ curl http://localhost:2224/metrics -s  | grep bpfstats
+        mapCount{gadgetImage="trace_exec:latest",gadgetName="mytraceexec",otel_scope_name="bpfstats",otel_scope_version="",progName=""} 7
+        mapCount{gadgetImage="trace_open:latest",gadgetName="mytraceopen",otel_scope_name="bpfstats",otel_scope_version="",progName=""} 6
+        mapMemory{gadgetImage="trace_exec:latest",gadgetName="mytraceexec",otel_scope_name="bpfstats",otel_scope_version="",progName=""} 1.83170336e+08
+        mapMemory{gadgetImage="trace_open:latest",gadgetName="mytraceopen",otel_scope_name="bpfstats",otel_scope_version="",progName=""} 1.71024e+06
+        otel_scope_info{otel_scope_name="bpfstats",otel_scope_version=""} 1
+        runcount_total{gadgetImage="trace_exec:latest",gadgetName="mytraceexec",otel_scope_name="bpfstats",otel_scope_version="",progName=""} 636
+        runcount_total{gadgetImage="trace_open:latest",gadgetName="mytraceopen",otel_scope_name="bpfstats",otel_scope_version="",progName=""} 219513
+        runtime_total{gadgetImage="trace_exec:latest",gadgetName="mytraceexec",otel_scope_name="bpfstats",otel_scope_version="",progName=""} 445114
+        runtime_total{gadgetImage="trace_open:latest",gadgetName="mytraceopen",otel_scope_name="bpfstats",otel_scope_version="",progName=""} 7.3383934e+07
+        ```
+    </TabItem>
+</Tabs>
+
+Finally, stop metrics collection:
+
+<Tabs groupId="env">
+    <TabItem value="kubectl-gadget" label="kubectl gadget">
+        ```bash
+        WIP: Headless mode for kubectl gadget is under development
+        ```
+    </TabItem>
+
+    <TabItem value="ig" label="ig">
+        ```bash
+        $ sudo gadgetctl delete 8423cd8e53339c8d4501ec7cdff436bc
+        8423cd8e53339c8d4501ec7cdff436bc
+        ```
+    </TabItem>
+</Tabs>
+
+### Limitations
+
+#### Memory usage of maps
+
+The shown value for MapMemory is read from `/proc/<pid>/fdinfo/<map_id>`. This
+is the maximum size the map can have, but it doesn't necessarily reflect its
+current memory allocation. Additionally, maps can be used by more than one
+program and would account towards the MapMemory of all those programs.
+
+Also note:
+* BPF_MAP_TYPE_PERF_EVENT_ARRAY: value_size is not counting the ring buffers,
+  but only their file descriptors (i.e. sizeof(int) = 4 bytes)
+* BPF_MAP_TYPE_{HASH,ARRAY}_OF_MAPS: value_size is not counting the inner maps,
+  but only their file descriptors (i.e. sizeof(int) = 4 bytes)
+
+#### bpfstats not disabled for Kernels < 5.8
+
+The bpfstats Gadgets enables collection of eBPF program stats by trying to use
+`BPF_ENABLE_STATS` first (which requires Linux >= 5.8). If that fails, it will
+fall back to trying to enable via sysctl (/proc/sys/kernel/bpf_stats_enabled),
+on this case,  the collection of stats will remain active even after stopping
```

Comment:
nit:

```suggestion
on this case, the collection of stats will remain active even after stopping
```

WDYT about specifying:
1. This is a temporary limitation. Just to let people know we are working on it.
2. That they can stop it by running: `sysctl kernel.bpf_stats_enabled=0`.